### PR TITLE
[template-utils] 📦 🛢 Use MarkdownField from sfdo_template_helpers

### PR DESCRIPTION
**Pull request by wlonk**
squirrel

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -28,3 +28,4 @@ django-hashid-field==2.1.5
 redis>=2.10,<3.0  # pyup: >=2.10,<3.0
 markdown==3.0.1
```
Comment (jgerigmeyer) [inline requirements/base.txt]: @wlonk Can we remove markdown and bleach here?

File: requirements/base.txt [hunk 2]
```diff
@@ -28,3 +28,4 @@ django-hashid-field==2.1.5
...
 logfmt==0.4
+-e git+https://github.com/SFDO-Tooling/sfdo-template-helpers.git@4765a7db3793fad75a3f1158de13e2fca23cb7e8#egg=sfdo_template_helpers
```
Comment (jgerigmeyer) [inline requirements/base.txt]: Should this be a release, either on PyPi or at least a git release/tag?

File: requirements/base.txt (hunk 1 above)
Comment (jgerigmeyer) [reply requirements/base.txt]: (yes to markdown, no to bleach)

**Comment by wlonk** [comment]
@jgerigmeyer ready to merge when tests pass and you 👍 it.

**Pull request merged**